tests/test_boundary_layer.py

```diff
@@ -21,3 +21,11 @@ def test_mixed_layer_height_RHmax(ds_isentropic_test_profiles):
         ds=ds, altitude="alt", rh="rh"
     )
     assert np.allclose(da_rh_peak, z0)
+
+    import numpy as np
+
+def test_inversion_height_gradient_RH(ds_isentropic_test_profiles):
```

Comment:
Awesome ! 🌟

Only there's a `import numpy as np` flying around there. Are you using that? :)

Reply:
Mmh ja that shouldn't be there...